modules/3d-tiles/src/lib/parsers/helpers/normalize-3d-tile-colors.ts

```diff
@@ -1,13 +1,19 @@
+import {Tile3DBatchTable} from '@loaders.gl/3d-tiles';
 import {decodeRGB565, GL} from '@loaders.gl/math';
+import {Tiles3DTileContent} from 'modules/3d-tiles/src/types';
```

Comment:
Please import only from relative files (or if outside of module, from scoped module roots `@loaders.gl/3d-tiles`)